sdk/postgresql/Azure.ResourceManager.PostgreSql/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
   "TagPrefix": "net/postgresql/Azure.ResourceManager.PostgreSql",
-  "Tag": "net/postgresql/Azure.ResourceManager.PostgreSql_3058e2076a"
+  "Tag": "net/postgresql/Azure.ResourceManager.PostgreSql_1f4154b2f1"
```

Comment:
what is this Tag for? How is it different from TagPrefix above?

Reply:
Previous tag referenced old recordings. New tag was created once pushing new recordings to asset repo.

[test guide](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#test-proxy)